internal/operators/cnv/cnv_operator_test.go

```diff
@@ -352,29 +349,37 @@ var _ = Describe("CNV operator", func() {
 	})
 
 	Context("cluster requirements", func() {
-		It("only x86_64 is supported for CNV operator", func() {
-			cluster := common.Cluster{}
+		cluster := common.Cluster{}
+		table.DescribeTable("Validate Cluster", func(ocpVersion []string, cpuArch string, expectedApiStatus api.ValidationStatus) {
+			cluster.CPUArchitecture = cpuArch
+			for _, v := range ocpVersion {
+				cluster.OpenshiftVersion = v
+				validation, err := operator.ValidateCluster(context.TODO(), &cluster)
+				Expect(err).ToNot(HaveOccurred())
 
-			cluster.CPUArchitecture = common.DefaultCPUArchitecture
-			validation, err := operator.ValidateCluster(context.TODO(), &cluster)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(validation.Status).To(Equal(api.Success))
+				Expect(validation.Status).To(Equal(expectedApiStatus),
 
-			cluster.CPUArchitecture = "arm64"
-			validation, err = operator.ValidateCluster(context.TODO(), &cluster)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(validation.Status).To(Equal(api.Failure))
-			Expect(validation.Reasons).To(ContainElements(
-				"OpenShift Virtualization is supported only for x86_64 CPU architecture."))
```

Comment:
This error message still exist in the `cnv_operator.validateRequirements`. It should be fixed.
```
func (o *operator) validateRequirements(cluster *models.Cluster) (api.ValidationStatus, string) {
	if !featuresupport.IsFeatureCompatibleWithArchitecture(models.FeatureSupportLevelIDCNV, cluster.OpenshiftVersion, cluster.CPUArchitecture) {
		return api.Failure, fmt.Sprintf(
			"%s is supported only for %s CPU architecture.", o.GetFullName(), common.DefaultCPUArchitecture)
	}
	return api.Success, ""
}
```